src/util/dom-repeat-once.html

```diff
@@ -0,0 +1,34 @@
+<dom-module id="dom-repeat-once">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <content></content>
+
+  </template>
+  <script>
+    Polymer({
+      is: 'dom-repeat-once',
+      properties: {
+        items: Array
+      },
+      reset: function(getItemElem) {
+        if (this.items && this.items.length > 0) {
+          this.items.sort(function(a, b) {
+              if(!b.order || a.order < b.order) return -1;
+              if(!a.order || a.order > b.order) return 1;
+              return 0;
+          });
+
+          this.innerHTML = '';
```

Comment:
It is faster to remove the first child. See http://stackoverflow.com/a/3955238/2761676